app/src/main/java/uk/ac/ebi/atlas/utils/ExpressionAtlasBioentityIdentifiersReader.java

```diff
@@ -185,11 +190,26 @@ private int addBioentityIdentifiersFromRnaSeqDifferentialExperiments(HashSet<Str
                     }
                 } catch (IOException exception) {
                     LOGGER.error(exception.getMessage());
+                    if (throwError) {
+                        throw exception;
+                    }
                 }
             }
 
         }
 
         return bioentityIdentifiers;
     }
+
+    @Override
+    public HashSet<String> getBioentityIdsFromExperiment(String experimentAccession) {
+        HashSet<String> bioentityIds = new HashSet<>();
+        try {
+            return getBioentityIdsFromExperiment(experimentAccession, false);
+        } catch (Exception e) {
+            // this will never be reached.
+            LOGGER.error("This should have not been reached:" + e.getMessage());
+        }
+        return bioentityIds;
+    }
```

Comment:
I had to do this because that new signature (with the boolean) has now a throw, so it would mean that the previous method would require a throw as well (breaking the previous usage). But I could imagine that there is a cleaner way of doing this?

Reply:
Oh, I see. My choice would be not to throw, and do the `UncheckedIOException` I suggested above.

Reply:
Done it, thanks!